sdks/python/setup.py

```diff
@@ -120,121 +120,11 @@ def get_version():
 except ImportError:
   cythonize = lambda *args, **kwargs: []
 
-REQUIRED_PACKAGES = [
-    # Avro 1.9.2 for python3 was broken. The issue was fixed in version 1.9.2.1
-    'crcmod>=1.7,<2.0',
-    # dataclasses backport for python_version<3.7. No version bound because this
-    # is Python standard since Python 3.7 and each Python version is compatible
-    # with a specific dataclasses version.
-    'dataclasses;python_version<"3.7"',
-    'orjson<4.0',
-    # Dill doesn't have forwards-compatibility guarantees within minor version.
-    # Pickles created with a new version of dill may not unpickle using older
-    # version of dill. It is best to use the same version of dill on client and
-    # server, therefore list of allowed versions is very narrow.
-    # See: https://github.com/uqfoundation/dill/issues/341.
-    'dill>=0.3.1.1,<0.3.2',
-    'cloudpickle>=2.0.0,<3',
-    'fastavro>=0.23.6,<2',
-    'grpcio>=1.29.0,<2',
-    'hdfs>=2.1.0,<3.0.0',
-    'httplib2>=0.8,<0.20.0',
-    'numpy>=1.14.3,<1.23.0',
-    'pymongo>=3.8.0,<4.0.0',
-    'protobuf>=3.12.2,<4',
-    'proto-plus>=1.7.1,<2',
-    'pyarrow>=0.15.1,<8.0.0',
-    'pydot>=1.2.0,<2',
-    'python-dateutil>=2.8.0,<3',
-    'pytz>=2018.3',
-    'requests>=2.24.0,<3.0.0',
-    'typing-extensions>=3.7.0',
-]
-
 # [BEAM-8181] pyarrow cannot be installed on 32-bit Windows platforms.
+pyarrow_dependency = 'pyarrow>=0.15.1,<8.0.0'
+pyarrow_dependency = ''
```

Comment:
Is this line correct? Also, prefer go/python-tips/026

Reply:
Whoops, I wanted to quickly test that this didn't break things and forgot to remove - thanks! Fixed